app/assets/stylesheets/_form.sass

```diff
@@ -13,6 +12,10 @@ form
   margin: -200px 0 0 -400px
   overflow: hidden
   height: 500px
+  p 
+    font-weight: 600
+    color: #555
+    font-size: 27px
   form
```

Comment:
Can we reference a variable here? Instead of @555, I bet we will use this other places.

Also, is that text a p tag? Maybe it should be an h2 or something. I like to reserve p tags for body text only